packages/grid/data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -176,11 +174,16 @@ describe('<DataGrid /> - Keyboard', () => {
     expect(isSelected).to.equal(true);
   });
 
-  it('Space only should go to the bottom of the page', () => {
+  it('Space only should go to the bottom of the page', function test() {
+    if (/jsdom/.test(window.navigator.userAgent)) {
```

Comment:
why do we need to retest that's jsDom here?

Reply:
I skip this test because I can't get it pass with both layouting and no layouting available.

Reply:
I meant
```tsx
if (isJsDOM)) {
      // Need layouting for row virtualization
      this.skip();
}
```

Reply:
It looks like the same. I can create an NO_LAYOUT to abstract the why.